### DISCUSSION TITLE
How to remove cookie value?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17866] [Questions] [answered]
How to remove cookie value

`useCookie('token').value = '123'`
--- and now i want to remove this value
`useCookie('token').value = ''`

**Comment by wobsoriano** [comment, marked as answer]
Just set the value to `null` or `undefined`

**Reply by mihail727** [reply]
> Just set the value to `null` or `undefined`

I just thought there might be a straight forward solution
